app/assets/stylesheets/admin.scss

```diff
@@ -9,8 +9,6 @@
  * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
- *= require settings/normalize
```

Comment:
@bitTal this was needed to avoid FE styling from affecting the backoffice interface. Please let me know if we should includo normalize/anything else for the backoffice styling at this point